File: .config/template/template.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
# FileName: 	<+FILENAME+>
# Author: 8ucchiman
# CreatedDate:  <+DATE+>
# LastModified: 2023-01-23 14:14:11 +0900
# Reference: 8ucchiman.jp
#


import os
import sys
import utils
# import numpy as np
# import pandas as pd


def main():
    args = utils.get_args()
    method = getattr(utils, args.method)
    <+CURSOR+>


if __name__ == "__main__":
    main()

File: .config/snippets/codes/c++/basics/template.cpp
/*
 * FileName:     template
 * Author: 8ucchiman
 * CreatedDate:  2023-03-23 22:39:59 +0900
 * LastModified: 2023-03-23 22:42:50 +0900
 * Reference: https://qiita.com/hal1437/items/b6deb22a88c76eeaf90c
 */

#include <iostream>


template<typename T>
T Add(T a, T b) {
    return a+b;
}

int main(void){
    std::cout << Add(2, 4) << std::endl;
    return 0;
}
